src/server/transaction.cc

```diff
@@ -1023,7 +1035,7 @@ KeyLockArgs Transaction::GetLockArgs(ShardId sid) const {
 // Returns true if was eagerly executed, false if it was scheduled into queue.
```

Comment:
is this comment is correct?
we eagerly execute `RunQuickie` but then we discover `AVOID_CONCLUDING`, so we lock the keys and insert. 
Probably the comment needs a fix.

Reply:
Right, needs an update